### PR TITLE
fix: Security services' bootstrap handler return true/false properly

### PR DESCRIPTION
**Pull request by lenny-intel**
Also updated to latest go mods which exposed this issue.

fixe #4106

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **N/A**
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?) **N/A**
- [ ] I have opened a PR for the related docs change (if not, why?) **N/A**
  <link to docs PR>

### CODE REVIEW

**Review by jim-wang-intel — DISMISSED** [review]
LGTM

**Review by bnevis-i — DISMISSED** [review]

**Review by jim-wang-intel — DISMISSED** [review]

**Review by jim-wang-intel — DISMISSED** [review]

**Comment by lenny-intel** [comment]
@farshidtz , can you look at the snap testing failure for this PR that should have resolved the non-zero exit.

File: internal/security/config/main.go [hunk 1]
```diff
@@ -51,19 +51,21 @@ func Main(ctx context.Context, cancel context.CancelFunc) int {
 
 	serviceHandler := NewBootstrap()
 
-	bootstrap.Run(
+	_, _, _ = bootstrap.RunAndReturnWaitGroup(
```
Comment (farshidtz) [inline internal/security/config/main.go:54]:
```suggestion
	bootstrap.RunAndReturnWaitGroup(
```

**Review by farshidtz — COMMENTED** [review]
Thanks for hunting down the issue @lenny-intel and @jim-wang-intel.

**Review by jim-wang-intel — APPROVED** [review]
LGTM

File: internal/security/config/main.go (hunk 1 above)
Comment (jim-wang-intel) [reply internal/security/config/main.go:54]: @farshidtz those empty returns to avoid Golang linting error

Comment (lenny-intel) [reply internal/security/config/main.go]: What thinking the linter would complain, but that is only if it is an error message. Removed.

Comment (jim-wang-intel) [reply internal/security/config/main.go]: okay, good to know thanks.

**Review by jim-wang-intel — APPROVED** [review]

**Review by bnevis-i — APPROVED** [review]

**Pull request merged**